src/workerd/api/global-scope.h

```diff
@@ -176,10 +176,15 @@ class ExecutionContext: public jsg::Object {
 public:
   void waitUntil(kj::Promise<void> promise);
   void passThroughOnException();
+  void abort(jsg::Lock& js, jsg::Optional<jsg::Value> reason);
```

Comment:
nit: some documentation comments here explaining the semantics expected would be helpful

Reply:
b17aa75c